packages/ui-app/src/util/keyboard.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { KEYS } from '../constants';
+
+const regex = {
+  decimalPoint: /[\.]/gi
+};
+
+// obtain current cursor index position of an input field
+const cursorIndexInputField = (event: any) => event.target.value.slice(0, event.target.selectionStart).length;
+
+const isNotDigit = (event: any): boolean => (event.keyCode < 48 || event.keyCode > 57);
```

Comment:
For the whole file, split the definition of the function from the actual body. (See readability improvement comments earlier)